turbinia/task_manager.py

```diff
@@ -251,6 +255,78 @@ def run(self):
       time.sleep(10)
 
 
+class CeleryTaskManager(BaseTaskManager):
+  """Celery implementation of BaseTaskManager.
+
+  Attributes:
+    celery (TurbiniaCelery): Celery task queue, handles worker tasks.
+    kombu (TurbiniaKombu): Kombu queue, handles receiving evidence.
+  """
+
+  def __init__(self):
+    self.celery = None
+    self.kombu = None
+    config.LoadConfig()
+    super(CeleryTaskManager, self).__init__()
+
+  def _backend_setup(self):
+    self.celery = turbinia_pubsub.TurbiniaCelery()
+    self.celery.setup()
+    self.kombu = turbinia_pubsub.TurbiniaKombu(config.KOMBU_CHANNEL)
+    self.kombu.setup()
+
+  def process_tasks(self):
+    """Determine the current state of our tasks.
+
+    Returns:
+      list[TurbiniaTask]: all completed tasks
+    """
+    completed_tasks = []
+    for task in self.tasks:
+      celery_task = task.stub
+      if not celery_task:
+        log.debug('Task {0:s} not yet created'.format(task.stub.task_id))
+      elif celery_task.status == celery_states.STARTED:
+        log.debug('Task {0:s} not finished'.format(celery_task.id))
+      elif celery_task.status == celery_states.FAILURE:
+        log.warning('Task {0:s} failed.'.format(celery_task.id))
+        completed_tasks.append(task)
+      elif celery_task.status == celery_states.SUCCESS:
+        task.result = celery_task.result
+        completed_tasks.append(task)
+      else:
+        log.debug('Task {0:s} status unknown'.format(celery_task.id))
+
+    outstanding_task_count = len(self.tasks) - len(completed_tasks)
+    log.info('{0:d} Tasks still outstanding.'.format(outstanding_task_count))
+    # pylint: disable=expression-not-assigned
```

Comment:
I guess this can get removed.  I probably forgot to remove it in the PSQ object after changing the return not to be a list comprehension.